verification/curator-service/ui/package.json

```diff
@@ -26,7 +27,7 @@
   },
   "scripts": {
     "start-noenv": "react-scripts start",
-    "start": "PORT=3002 REACT_APP_API_ENDPOINT=http://localhost:3001 react-scripts start",
+    "start": "PORT=3002 react-scripts start",
```

Comment:
If you're in the business of `|| <default>` for the API endpoint, do you want to just do the same for the port?